documentation/modules/ROOT/pages/connectors/mongodb.adoc

```diff
@@ -1591,6 +1574,16 @@ The following configuration properties are _required_ unless a default value is
 |===
 |Property |Default |Description
 
+|[[mongodb-property-allow-offset-invalidation]]<<mongodb-property-allow-offset-invalidation, `+internal.mongodb.allow.offset.invalidation+`>>
+|false
+|Allows shard specific offsets recorded by older connector versions to be invalidated and xref:mongodb-offset-consolidation[consolidated]
+
+[WARNING]
+====
+
+This property is considered internal, it will be removed and the default behaviour will be as if enabled in future releases.
```

Comment:
```suggestion
This is an internal property, and is scheduled for removal.

ifdef::community[]
Subsequent releases of the connector will invalidate and consolidate shard-specific offsets.  
endif::community[]
```
For the downstream docs, I think that it's ok to "pre-announce" removal of the property, but we should not make statements that describe possible future behavior.

Reply:
Lets hear @jpechane here as this is somwewhat special case. This property was introduced as temporary. It's purpose is only for users to transition from one model to the other.

Reply:
I think the text should go to the downstream docs as well. If @roldanbob is not fully on-board wit definitive statement I think it could be qualified like "It is expected that subsequent releases" or something like that signals high probability but not 100 % certainity.

Reply:
OK, I think that I've come up with wording that might satisfy. WDYT?